pkg/cache/cache_test.go

```diff
@@ -1496,6 +1529,20 @@ func TestCacheQueueOperations(t *testing.T) {
 			if diff := cmp.Diff(tc.wantQueueCounts, qCounts); diff != "" {
```

Comment:
Can you be more explicit?  I'm not entirely sure what line numbers you want me to remove.  

So what I am planning on doing:

I'd like to keep the call to cache.AdmittedWorkloadsLocalQueue because that is a public function of cache.  But I will remove the flattenedQueueCounts and just test the values of the map in wantQueueCounts with the values of AdmittedWorkloadsLocalQueue.

Reply:
Now that we have a public method `AdmittedWorkloadsLocalQueue`, there is no need to check the values of the internal map `cq.admittedWorkloadsPerQueue`, so we can remove these checks.

And you can flatten `wantQueueCounts` as `map[string]int32`

Reply:
Alright.  I added what you suggested.